File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        String[] can = {"aya", "ye", "woo", "ma"};
        
        int result = 0;

        for (int i = 0; i < babbling.length; i++) {
            for (int j = 0; j < can.length; j++) {
                if(babbling[i].contains(can[j])){
                    babbling[i] = babbling[i].replace(can[j]," ");
                }
            }
            babbling[i] = babbling[i].replace(" ","");
            if(babbling[i].length() == 0){
                result++;
            }
        }
        
        return result;
    }
}

File: 프로그래머스/unrated/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var answer = 0;
    for (let element of babbling) {
        element = element.replace('aya','.').replace('ye','.').replace('woo','.').replace('ma','.');
        const regex = /[a-zA-Z]+/;
        if(!regex.test(element)){
            answer++;
        }
    }
    return answer;
}